<?xml    version="1.0"    encoding="UTF-8"?>
<ui    version="4.0">
    <class>MainWindow</class>
    <widget    class="QMainWindow"    name="MainWindow">
        <property    name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>800</width>
                <height>800</height>
            </rect>
        </property>
        <property    name="windowTitle">
            <string>NiChartGUI</string>
        </property>
        <widget    class="QWidget"    name="centralwidget">
            <layout  class="QVBoxLayout"  name="vLayout">

                <item>
                    <widget    class="QTabWidget"    name="tabWidget">
                        <property    name="sizePolicy">
                            <sizepolicy    hsizetype="Expanding"    vsizetype="Expanding">
                                <horstretch>0</horstretch>
                                <verstretch>16</verstretch>
                            </sizepolicy>
                        </property>
                        <property    name="minimumSize">
                            <size>
                                <width>800</width>
                                <height>500</height>
                            </size>
                        </property>                        
                        <property    name="currentIndex">
                            <number>-1</number>
                        </property>
                    </widget>
                </item>

                <item>
                    <widget class="QTextEdit" name="wInfo">
                        <property    name="sizePolicy">
                            <sizepolicy    hsizetype="Expanding"    vsizetype="Expanding">
                                <horstretch>0</horstretch>
                                <verstretch>4</verstretch>
                            </sizepolicy>
                        </property>
                        <property    name="maximumSize">
                            <size>
                                <width>20000</width>
                                <height>150</height>
                            </size>
                        </property>
                    </widget>
                </item>
                
            </layout>
        </widget>
        
        <widget    class="QMenuBar"    name="menubar">
            <property    name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>800</width>
                    <height>22</height>
                </rect>
            </property>
            <widget    class="QMenu"    name="menuFile">
                <property    name="title">
                    <string>File</string>
                </property>
                <addaction name="actionOpenData"/>
                <addaction name="actionSaveData"/>
<!--                 <addaction name="actionSaveNotebook"/> -->
            </widget>
            <widget    class="QMenu"    name="menuPlugins">
                <property    name="title">
                    <string>Plugins</string>
                </property>
                
            </widget>
            <widget    class="QMenu"    name="menuHelp">
                <property    name="title">
                    <string>Help</string>
                </property>
                <addaction name="actionHelpConsole"/>
                <addaction name="separator"/>
                <addaction name="actionAbout"/>
            </widget>
            <addaction name="menuFile"/>
            <addaction name="menuPlugins"/>
            <addaction name="menuHelp"/>                        
        </widget>
        
        <widget    class="QStatusBar"    name="statusbar"/>
        
        <action name="actionP1">
            <property    name="text">
                <string>P1</string>
            </property>
        </action>
        <action name="actionOpenData">
            <property    name="text">
                <string>Load Dataset</string>
            </property>
        </action>
        <action name="actionSaveData">
            <property    name="text">
                <string>Save Dataset</string>
            </property>
        </action>
<!--        <action name="actionSaveNotebook">
            <property    name="text">
                <string>Create Notebook</string>
            </property>
        </action>-->
        <action name="actionAbout">
            <property    name="text">
                <string>About</string>
            </property>
        </action>
        <action name="actionHelpConsole">
            <property    name="text">
                <string>Help Console</string>
            </property>
        </action>
    </widget>
    <resources/>
    <connections/>
</ui>
